docs/docs/userguide/getting-started.md

```diff
@@ -0,0 +1,103 @@
+      /_/
+
+Using Scala version 2.10.5 (Java HotSpot(TM) 64-Bit Server VM, Java 1.7.0_79)
+Spark context available as sc.
+scala> 
+```
+
+
+For instance, to experiment with the ````Tensor```` APIs in BigDL, you may then try:
+```scala
+scala> import com.intel.analytics.bigdl.tensor.Tensor
+import com.intel.analytics.bigdl.tensor.Tensor
+
+scala> Tensor[Double](2,2).fill(1.0)
+res9: com.intel.analytics.bigdl.tensor.Tensor[Double] =
+1.0     1.0
+1.0     1.0
+[com.intel.analytics.bigdl.tensor.DenseTensor of size 2x2]
+```
+For more details about the BigDL APIs, please refer to the [Programming Guide](https://github.com/intel-analytics/BigDL/wiki/Programming-Guide).
+
+## Spark Program
+You can run a BigDL program, e.g., the [VGG](https://github.com/intel-analytics/BigDL/tree/master/spark/dl/src/main/scala/com/intel/analytics/bigdl/models/vgg) training, as a standard Spark program (running in either local mode or cluster mode) as follows:
+
+1. Download the CIFAR-10 data from [here](https://www.cs.toronto.edu/%7Ekriz/cifar.html). Remember to choose the binary version.
+
+2. Use the `bigdl.sh` script to launch the example as a Spark program as follows:
+
+```
+  # Spark local mode
+  ./dist/bin/bigdl.sh -- \
+  spark-submit --master local[core_number] --class com.intel.analytics.bigdl.models.vgg.Train \
+  dist/lib/bigdl-VERSION-jar-with-dependencies.jar \
+  -f path_to_your_cifar_folder \
+  -b batch_size
+
+  # Spark standalone mode
+  ./dist/bin/bigdl.sh -- \
+  spark-submit --master spark://... --executor-cores cores_per_executor \
+  --total-executor-cores total_cores_for_the_job \
+  --class com.intel.analytics.bigdl.models.vgg.Train \
+  dist/lib/bigdl-VERSION-jar-with-dependencies.jar \
+  -f path_to_your_cifar_folder \
+  -b batch_size
+
+  # Spark yarn mode
+  ./dist/bin/bigdl.sh -- \
+  spark-submit --master yarn --deploy-mode client \
+  --executor-cores cores_per_executor \
+  --num-executors executors_number \
+  --class com.intel.analytics.bigdl.models.vgg.Train \
+  dist/lib/bigdl-VERSION-jar-with-dependencies.jar \
+  -f path_to_your_cifar_folder \
+  -b batch_size
+```
+
+  The parameters used in the above command are:
+
+  * -f: The folder where your put the CIFAR-10 data set. Note in this example, this is just a local file folder on the Spark driver; as the CIFAR-10 data is somewhat small (about 120MB), we will directly send it from the driver to executors in the example.
+
+  * -b: The mini-batch size. The mini-batch size is expected to be a multiple of *total cores* used in the job. In this example, the mini-batch size is suggested to be set to *total cores * 4*
+
+## Next Steps
+* To learn the details of Python support in BigDL, you can check out the [Python Support Page](https://github.com/intel-analytics/BigDL/wiki/Python-Support)
+* To learn how to create practical neural networks using BigDL in a couple of minutes, you can check out the [Tutorials Page](https://github.com/intel-analytics/BigDL/wiki/Tutorials)
```

Comment:
Hyper links to wiki pages needs to be modified. Wiki pages will be removed later.